### DISCUSSION TITLE
Quick Help for "Path_to_HuggingFace"

### DISCUSSION THREAD

**Discussion by metzo007** [TheLastBen/fast-stable-diffusion#1492] [Q&A]
Hi,

can someone tell me how to get the Path_to_HuggingFace for Protogen 2.2?

https://huggingface.co/darkstorm2150/Protogen_v2.2_Official_Release/tree/main

**Comment by TheLastBen** [comment]
`darkstorm2150/Protogen_v2.2_Official_Release`

**Reply by metzo007** [reply]
thank you sir :-)
